### PR TITLE
[DOC-2401] Update access-control-model.adoc for adding built-in role globalobserver;

### PR DESCRIPTION
**Pull request by zhihuapengtg**

### CODE REVIEW

**Comment by zhihuapengtg** [comment]
@arun-TG-PM @victorleeTG 
please help to review it when you have time, thanks!

File: modules/user-access/pages/access-control-model.adoc [hunk 1]
```diff
@@ -303,6 +303,10 @@ The following table details the built-in roles and their corresponding set of pr
 `WRITE_POLICY`,
 `OWNERSHIP` on ALL QUERIES
 
+| `globalobserver`
+| Global
+| `READ_SCHEMA`, `READ_LOADINGJOB` on global scope
```
Comment (dadongwang-tg) [inline modules/user-access/pages/access-control-model.adoc]: "on global scope" -> "on the global scope"

File: modules/user-access/pages/access-control-model.adoc [hunk 2]
```diff
@@ -303,6 +303,10 @@ The following table details the built-in roles and their corresponding set of pr
 `WRITE_POLICY`,
 `OWNERSHIP` on ALL QUERIES
 
+| `globalobserver`
+| Global
+| `READ_SCHEMA`, `READ_LOADINGJOB` on the global scope
```
Comment (victorleeTG) [inline modules/user-access/pages/access-control-model.adoc]:
```suggestion
| `READ_SCHEMA`, `READ_LOADINGJOB` at the global scope
```

**Review by victorleeTG — APPROVED** [review]
Changed "on the global scope" to "at the global scope"

**Pull request merged**